app/components/PageWrapper.js

```diff
@@ -29,11 +30,13 @@ class PageWrapper extends Component {
   componentDidMount() {
     ipcRenderer.on('update-downloaded', this.onAppUpgrade);
     ipcRenderer.on('play-replay', this.onPlayReplay);
+    ipcRenderer.on('play-local-replay', (event, slppath) => { this.onPlayLocalReplay(slppath); });
   }
 
   componentWillUnmount() {
     ipcRenderer.removeListener('update-downloaded', this.onAppUpgrade);
     ipcRenderer.removeListener('play-replay', this.onPlayReplay);
+    ipcRenderer.removeListener('play-local-replay', (event, slppath) => { this.onPlayLocalReplay(slppath); });
```

Comment:
Does removeListener work if you pass it a different anonymous function definition?

Reply:
I dunno. I just kinda copied how you wrote the URI handler. It never complained when I tested it.